roscala/src/test/scala/coop/rchain/rosette/TransitionSpec.scala

```diff
@@ -524,4 +524,61 @@ class TransitionSpec extends FlatSpec with Matchers {
     end.ctxt.ctxt.trgt shouldBe Fixnum(100)
   }
 
+  "OpRtnArg" should "copy ctxt.rslt to given argvec element" in {
+
+    /**
+      * This opcode sequence will never be generated by the RBL
+      * compiler as it is the responsibility of the caller to
+      * specify where the result will be stored.
+      *
+      * litvec:
+      *   0:   100
+      * codevec:
+      *   0:   liti 0,rslt
+      *   1:   rtn/nxt arg[1]
+      */
+    val start =
+      testState
+        .set(_ >> 'ctxt >> 'ctxt)(testState.ctxt)
+        .set(_ >> 'ctxt >> 'argvec)(Tuple(Seq(Ob.NIV, Ob.NIV, Ob.NIV)))
```

Comment:
Can you explain what this line is serving for?

Reply:
`OpRtnArg(next = true, arg = 1)` writes to `state.ctxt.argvec.elem(1)` which needs to exist.